### ISSUE TITLE
Schwarm-Mobs in Silithus oft nicht loot/kürschnerbar

### ISSUE THREAD

**Issue by grindbox** [ClassicWoW/Nefarian_1.12.1_Bugtracker#1682]
Welches Verhalten wird beobachtet?

wenn man getötete Silithiden kürschnern will, kommt die meldung "Die Kreatur muss zuerst geplündert werden" obwohl beim mouseover das kürschnerzeichen erscheint. Tritt  immer auf wenn der mob vornerein nix drin hatte, deshalb kann man ihn nich "leerlooten" weil er schon leer ist.

Wie sollte es sich verhalten?
Wenn der Mob nix zum looten hat, sollte normales Kürschnern möglich sein.

Schritte zur Reproduzierung

Zusätzliche Informationen (Screenshots, Videos, etc.)
Beispiel-Mob http://datenbank.classic-wow.org/?npc=11722
[url=http://www.fotos-hochladen.net][img]http://img5.fotos-hochladen.net/uploads/wowscrnshot0800pkvjlum4c.jpg[/img][/url]

Alle Kreaturen, Items, Objekte, Quests, Zauber etc. müssen aus unserer Datenbank verlinkt sein.

**Comment by Loplin** [comment]
Hast du versucht das kürschnern-symbol aus dem spellbook zu benutzen?
Möglicherweise funktioniert das Kürschnern dann
Falls ja wäre das natürlich nur eine möglichkeit das problem zu umgehen

**Comment by grindbox** [comment]
konnt ich bis jetz nich testen. wenn ich soll unterwegs bin geht alles, aber in der gruppe tritt das problem auf.

**Comment by grindbox** [comment]
Hab jetzt nochmal getestet und der Bug tritt auch ohne Grp auf. mit dem kürschner-symbol klappts leider auch nicht.

**Comment by hraesvelgr** [comment]
besitzt du dieses item, um silithidenschuppen zu sammeln, und wenn ja: hast du es zufällig auf der bank?

**Comment by grindbox** [comment]
welches item soll das sein? sammle silithidenschuppen auch so, der bug tritt halt nur manchmal auf

**Comment by hraesvelgr** [comment]
die silithid carapce fragments kann man nur sammeln wenn man eines dieser items besitzt http://datenbank.classic-wow.org/?search=nozd#items
evtl besitzt du die, hast sie aber auf der bank?!

**Comment by grindbox** [comment]
nein, aber mir wurde grad gesagt, dass das auch vorkommt wenn irgendein anderer spieler als agent des nozdormus item auf der bank liegen hat und deshalb andere die mobs nich looten können weil des questitem im mob nich zusehen ist
